changelogs/fragments/connection_reset.yaml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+- ssh - reset connection will show a warning instead of failing when resetting the ControlPath for older OpenSSH versions
```

Comment:
s/ControlPath/connection/ ?

Reply:
easily changed, what do you mean by "more polling on the `exit` fallback"?